src/cr/cube/matrix.py

```diff
@@ -1899,6 +1904,11 @@ def proportions(self):
     def table_margin(self):
         return self._table_margin
 
+    @lazyproperty
+    def unweighted_counts(self):
+        """1D ndarray of unweighted count for each valid element in vector."""
```

Comment:
Ok, I see a pattern now, I'd like an explanation of why we want to use "of unweighted count" instead of "of unweighted counts" :)